samples/client/petstore/typescript-aurelia/default/PetApi.ts

```diff
@@ -19,6 +19,7 @@ import {
   any,
   Pet,
   ApiResponse,
+  Array&lt;&#39;available&#39; | &#39;pending&#39; | &#39;sold&#39;&gt;,
```

Comment:
it seems that this was already broken before, see the previous lines:
```
 import {
   Array&lt;string&gt;,		  
```

Reply:
@macjohnny what about fixing it by replacing {{ ... }} with  {{{ ... }}} ?

Reply:
@wing328 it is broken even if the html issue is fixed, since it tries to import ‚Array<string>‘ from the file, but this is a generic type...
I will open a separate issue to have this fixed.

Reply:
see https://github.com/swagger-api/swagger-codegen/issues/7465